webextensions/api/webRequest.json

```diff
@@ -396,6 +459,27 @@
               }
             }
           },
+          "other": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "44"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "45"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": false
```

Comment:
@queengooborg info here says Safari  supports this https://github.com/mdn/content/pull/22644#discussion_r1101963224